README.md

```diff
@@ -302,6 +303,11 @@ The Node API is structured similarly to the command line options, but there are
   * <a name="screenshot-capture-mode" href="#screenshot-capture-mode">#</a> **Screenshot capture mode** (default) uses puppeteer's built-in API to take screenshots of Chromium/Chrome windows. It can capture most parts of a webpage (e.g. div, svg, canvas) as they are rendered on the webpage. It can crop images, round to even widths/heights, but it usually runs slower than canvas capture mode.
   * <a name="canvas-capture-mode" href="#canvas-capture-mode">#</a> **Canvas capture mode** (experimental) directly copies data from a canvas element and is often faster than using screenshot capture mode. If the background of the canvas is transparent, it may show up as transparent or black depending on the captured image format. Configuration options that adjust the crop and round to an even width/height do not currently have an effect. To use this mode, [use the `--canvas-capture-mode` option from the command line](#cli-options-canvas-capture-mode) or [set `config.canvasCaptureMode` from Node.js](#js-config-canvas-capture-mode). Also specify the canvas using a css selector, [using the `--selector` option from the command line](#cli-options-selector) or [setting `config.selector` from Node.js](#js-config-selector), otherwise it uses the first canvas element.
 
+## <a name="controlFromPage" href="#controlFromPage">#</a> Control From Page
+The captured page can call window.stopCapture() to end the capture process before `--duration` or `--frames` is up. Useful if you don't know ahead of time how long a capture should take (set a long enough duration and call stopCapture when finished).
+
+Use `if (window.stopCapture) window.stopCapture();` to avoid javasript errors when page is not loaded via **timesnap**.
+
```

Comment:
```suggestion
```
This should be moved to the command line and JavaScript API sections (also change `javasript` to `JavaScript`)